types/emoji-mart/dist-es/components/picker/nimble-picker.d.ts

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
```

Comment:
Keep all React imports as `import React = require('react')`. As long as react is [exported with `export =`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/aeb446e0bfc0ae94268414cd005ae52881b0a5ca/types/react/index.d.ts#L44) this is the only way of importing it that is correct regardless of the `esModuleInterop` setting.

Reply:
Understood. Didn't know there was a reason behind this. Thanks for the info 😄